.github/workflows/add_release_label.yml

```diff
@@ -0,0 +1,62 @@
+name: Add release label to PR and linked issues when PR gets merged
+
+on:
+  pull_request:
+    branches:
+      - develop
+    types:
+      - closed
+
+jobs:
+  add_release_label:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+            fetch-depth: 0 # This is needed to checkout all branches
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Install Yarn
+        run: npm install -g yarn
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Get the next semver version
+        id: get-next-semver-version
+        run: |
+          # Get the highest version from release branches
+          VERSION_BRANCHES=$(git branch -r | grep -o 'Version-v[0-9]*\.[0-9]*\.[0-9]*' | grep -o '[0-9]*\.[0-9]*\.[0-9]*' | sort --version-sort | tail -n 1)
+
+          # Get the highest version from tags
+          VERSION_TAGS=$(git tag | grep -o 'v[0-9]*\.[0-9]*\.[0-9]*' | grep -o '[0-9]*\.[0-9]*\.[0-9]*' | sort --version-sort | tail -n 1)
+
+          # Get the version from package.json
+          VERSION_PACKAGE=$(node -p "require('./package.json').version")
+
+          # Compare versions and keep the highest one
+          HIGHEST_VERSION=$(printf "%s\n%s\n%s" "$VERSION_BRANCHES" "$VERSION_TAGS" "$VERSION_PACKAGE" | sort --version-sort | tail -n 1)
+
+          # Increment the minor version of the highest version found
+          NEXT_VERSION=$(echo $HIGHEST_VERSION | awk -F. -v OFS=. '{$2++; print}')
+
+          # Check if FORCE_NEXT_SEMVER_VERSION is defined and not empty
+          if [ ! -z "${{ vars.FORCE_NEXT_SEMVER_VERSION }}" ]
+          then
+            NEXT_VERSION=${{ vars.FORCE_NEXT_SEMVER_VERSION }}
+          fi
+
+          echo "NEXT_SEMVER_VERSION=${NEXT_VERSION}" >> $GITHUB_ENV
+
+      - name: Add release label to PR and linked issues
+        id: add_release_label_to_pr_and_linked_issues
+        env:
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
We use a variety of different personal access tokens, each for a different purpose with different permissions. We should update this secret name to describe its purpose or permission set (e.g. `LABELLING_TOKEN`)

Reply:
Nice, that's a much better name than what I suggested